mockserver/mockserver.go

```diff
@@ -63,23 +65,32 @@ func (h *MockServerHandler) GetBlockCount() (int64, error) {
 
 func (h *MockServerHandler) GetBlockHash(blockHeight int64) (*chainhash.Hash, error) {
 	// get chainHash of block with blockHeight
-	blockHash := h.DataStore.DataContent.BlockHeaders[blockHeight].BlockHash
-	return &blockHash, nil
+	for _, blockHeader := range h.DataStore.DataContent.BlockHeaders {
+		if blockHeader.Height == blockHeight {
```

Comment:
We should organize the blocks in a map, so we can quickly query.

Reply:
Updated, please check